notebooks/official/feature_store/sdk-feature-store.ipynb

```diff
@@ -0,0 +1,1261 @@
+{
```

Comment:
<p>This Colab --&gt; This notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/486/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>